tests/testthat/test_featsel_filters.R

```diff
@@ -72,3 +72,16 @@ test_that("randomForestSRC_var.select filter handles user choices correctly", {
     more.args = list("randomForestSRC_var.select" = c(method = "vh.imp"))))
   )
 })
+
+
+test_that("Ensemble filters can accept duplicate base methods", {
+  lda = makeLearner(cl = "classif.lda", id = "lda_class", predict.type = "response")
+  ff = filterFeatures(multiclass.task, 
+                      method = "E-mean", 
+                      base.methods = list("filter1" = "univariate.model.score", "filter2" = "univariate.model.score"),
+                      abs = 2,
+                      more.args = list("filter1" = list(perf.learner = lda), "filter2" = list(perf.learner = lda))
+       )
+  expect_equal(getTaskFeatureNames(ff), c("Petal.Length", "Petal.Width"))
```

Comment:
Could you also please check that the returned data frame has the right column names, i.e. the filter names are reflected correctly in the output? Ideally with different methods that return different features so it's obvious which one is which.

Reply:
Just to clarify, when you say "the returned data frame", which data frame are you referring to and which function is returning it? Thanks.

Reply:
Sorry, ignore that -- I thought it returned something different.